### PR TITLE
Fix doc mentioning encodeToBuilder (fixes #612)

### LINKED ISSUE #612: docs of Data.Aeson.Text.encodeToTextBuilder mention "encodeToBuilder"
but that does not seem to exist/be exposed (since 1.*)

i suppose that function would be `encodingToLazyByteString . toEncoding` ?

### PR DESCRIPTION
**Pull request by lspitzner**
is it too late to [ci skip] ? lets see.

### CODE REVIEW

**Comment by lspitzner** [comment]
dang it :p i never think of that before i press "commit".

File: Data/Aeson/Text.hs [hunk 1]
```diff
@@ -46,8 +46,9 @@ encodeToLazyText = LT.decodeUtf8 . encodingToLazyByteString . toEncoding
 -- embedded efficiently in a text-based protocol.
 --
 -- If you are going to immediately encode straight to a
--- 'L.ByteString', it is more efficient to use 'encodeToBuilder'
--- instead.
+-- 'L.ByteString', it is more efficient to use
+-- @'encodingToLazyByteString' . 'toEncoding'@ (lazy ByteString) or
```
Comment (phadej) [inline Data/Aeson/Text.hs]: that's is just `encode` from `Data.Aeson`.

Comment (lspitzner) [reply Data/Aeson/Text.hs]: eh.. right

**Comment by lspitzner** [comment]
better?

**Comment by phadej** [comment]
that's up to @bergmark, I'm just an observer :)

**Comment by bergmark** [comment]
I would add you as a maintainer if i could @phadej :) Bryan suggested we move the repo to the haskell org a while back, hopefully that will happen at some point...

Thanks @lspitzner!

**Comment by hvr** [comment]
@phadej @bergmark we'd need @bos to initiate the transfer, and I gave him all necessary github-privs for that when `text` was migrated to the Haskell org already

**Pull request merged**